### ISSUE TITLE
BOS EOS 與 attr embedding

### ISSUE THREAD

**Issue by ga2006084851** [THUcqb/KOBE#29]
不好意思我有個問題想要問，關於TItle的文字序列部分，有在前後加上BOS EOS等起始終止符嗎？

如果說有，那關於Title embedding的部分，如果要像這篇論文一樣對每個token加上attr embedding，

是否也包括加在BOS、EOS等token的embedding上了呢？ 或者略過它們純粹對Title文字的token加上

呢？

謝謝作者的熱心！

**Comment by qibinc** [comment]
Hi @ga2006084851 ,

Sorry for the late reply. It's great that you noticed such detail and I really appreciate your attention to our work. We added both BOS and EOS to the title. For the attr embedding, we added it to all token embeddings including BOS and EOS and didn't experiment with excluding BOS/EOS. We followed https://arxiv.org/pdf/1603.06155, in which the authors concat the attr embeddings to the EOS token.

Hope this helps.
